mods/cnc/rules/aircraft.yaml

```diff
@@ -125,6 +126,7 @@ ORCA:
 		Description: Helicopter Gunship with AG Missiles.\n  Strong vs Buildings, Tanks\n  Weak vs Infantry
 	Aircraft:
 		InitialFacing: 224
+		TurnToLand: true
```

Comment:
Can we move this and the initial facing to ^Helicopter?

Reply:
Done.